changelog/1.35.3.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "1.35.3"
+description: "Released on 11/14/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.35.3.
+
+### Features ✨
+
+There are no new features in 1.35.3
+
+### Bug fixes 🐛
+
+- infra: Fixed an issue where the hostname was set to `workspace` for CVM
+  workspaces irrespective of the workspace name.
+- helm: Fixed an issue where the helm install was not respecting the 
```

Comment:
this was included in the 1.36.0 release, but not in the changeling. I'll submit a separate PR.